powerhome/playbook
# [NUXE-50] Add landing page and getting started to docs

#### Screens

![screencap](https://p-a6FbDk.t4.n0.cdn.getcloudapp.com/items/YEuy4XQ0/Screenshot-2020-10-28T17:55:59.642Z.png?v=1aef3d4dca9c164bc98a30c1d116f039)

#### What does this PR do?

1. Adds a new landing page and getting started page to the Playbook docs/site.
2. Refreshes the nav with links to the getting started page and existing pages in Playbook (kits, tokens, utilities).
3. Conditionally renders page specific version of the sidebar and the nav based on the route.
4. Cleans and simplifies the `application.html.erb` layout, creates a separate layout for all `/kits` routes.
5. Adds a footer partial to the `application.html.erb` layout.
6. Rescope dark mode cookie to only affect kit pages.

#### Breaking Changes

No breaking changes.  All routes/views accounted for.

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/NUXE-50)

#### How to test this

Test Milano deploy to ensure everything behaves as expected and all routes link appropriately.

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
